setup/operators/install_operators.go

```diff
@@ -0,0 +1,50 @@
+package operators
+
+import (
+	"context"
+
+	"github.com/codeready-toolchain/toolchain-e2e/setup/resources"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/pkg/errors"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	AllNamespacesOperatorName   = "kiali-ossm"
+	AllNamespacesOperatorPrefix = "kiali-operator"
+)
+
+func EnsureAllNamespacesOperator(cl client.Client, csvNamespace string) error {
```

Comment:
Let's add a comment explaining why we install operators here.

Reply:
Sure, I had a comment in root.go but it's worth having one here too. Done 1bbdd9c38f46e6cad22e261315e7f2b837b72b90